src/Components/Contributors/ContributorSearchResultItem.tsx

```diff
@@ -78,38 +61,77 @@ const ContributorSearchResultItem: FC<ContributorSearchResultItemProps> = ({ con
-              title="Ukjent verifikasjonsstatus"
-            />
+              {`${contributor.first_name_preferred ?? contributor.first_name} ${
+                contributor.surname_preferred ?? contributor.surname
+              }`}
+            </StyledActivePersonNameTypography>
           ) : (
-            <StyledNotVerifiedBadge data-testid={`author-name-${contributor.cristin_person_id}-not-verified-badge`} />
+            <StyledInactivePersonNameTypography
+              data-testid={`author-name-${contributor.cristin_person_id}`}
+              variant="h6">
+              {contributor.require_higher_authorization ? (
+                <StyledUnknownVerifiedBadge
+                  data-testid={`author-name-${contributor.cristin_person_id}-uknown-verified-badge`}
+                  title="Ukjent verifikasjonsstatus"
+                />
+              ) : (
+                <StyledNotVerifiedBadge
+                  data-testid={`author-name-${contributor.cristin_person_id}-not-verified-badge`}
+                />
+              )}
+              {`${contributor.first_name_preferred ?? contributor.first_name} ${
+                contributor.surname_preferred ?? contributor.surname
+              }`}
+            </StyledInactivePersonNameTypography>
           )}
-          {`${contributor.first_name_preferred ?? contributor.first_name} ${
-            contributor.surname_preferred ?? contributor.surname
-          }`}
-        </StyledInactivePersonNameTypography>
-      )}
-      {contributor.affiliations?.map((affiliation, affiliationIndex) => (
-        <StyledAffiliationsWrapper
-          data-testid={`list-item-author-${contributor.cristin_person_id}-affiliations-${affiliation.cristinInstitutionNr}`}
-          key={affiliationIndex}>
-          {affiliation.institutionName}
-          {affiliation.units &&
-            affiliation.units.map((unit, unitIndex) => <div key={unitIndex}>&bull; {unit.unitName}</div>)}
-        </StyledAffiliationsWrapper>
-      ))}
+        </Grid>
+        <Grid item sm={4}>
+          <Button
+            data-testid={`add-only-person-${contributor.cristin_person_id}`}
+            onClick={() => handleChooseOnlyAuthor(contributor)}
+            size="small"
+            color="primary">
+            Velg kun person
+          </Button>
+        </Grid>
+      </Grid>
+      {contributor.affiliations
+        ?.sort((affiliationA, affiliationB) => {
+          if (affiliationA.institutionName && affiliationB.institutionName) {
+            return affiliationA.institutionName.localeCompare(affiliationB.institutionName);
+          }
+          return 0;
+        })
+        .map((affiliation, affiliationIndex) => (
+          <AffiliationDisplay
+            addAffiliationSuccessful={addAffiliationSuccessful}
+            handleAddAffiliationButtonClick={() => handleChooseOnlyAffiliation(affiliation)}
+            key={`${affiliation.cristinInstitutionNr ?? 0}-${affiliationIndex}`}
```

Comment:
den keyen er vel bare for react ? om man først bruker affiliationIndex trengs vel ikke den første delen ?

Reply:
Bruke index alene er jo en dårlig idé når man kan slette ting i lista, og institutionID kan være null. (akkurat som problemet med "mine ressurser" på DLR, hvis du husker problemet med thumbnaileren når man slettet ting i lista.